### PR TITLE
Roombas have a flat 1 second attack cooldown

### PR DESCRIPTION
**Pull request by Kharkov72**
4.5 - 18 damage every time it walked over you
with literally no cooldown
for the low low cost of a lighter and a fork

now that's impressive
[balance]

:cl:
 * rscdel: Roombas now have a flat one second attack cooldown

### CODE REVIEW

File: code/game/machinery/bots/cleanbot.dm [hunk 1]
```diff
@@ -381,31 +380,29 @@ text("<A href='?src=\ref[src];operation=oddbutton'>[src.oddbutton ? "Yes" : "No"
...
-			annoy(L)
+		annoy(L)
 		..()
 
 /obj/machinery/bot/cleanbot/proc/attack_cooldown()
 	coolingdown = TRUE
-	spawn(attack_cooldown)
-		coolingdown = FALSE
+	spawn(1 SECONDS)
+	coolingdown = FALSE
```
Comment (PaulRitter) [inline code/game/machinery/bots/cleanbot.dm]: i think you missed an intendation here

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: code/game/machinery/bots/cleanbot.dm (hunk 1 above)
Comment (Kharkov72) [reply code/game/machinery/bots/cleanbot.dm]: it works either way but it came to me in my sleep that I should probably fix it

**Comment by agentofzog2** [comment]
I feel like this should only apply to PAIbots instead of all the bots since they'll just be even less annoying now.

**Review by Kurfursten — APPROVED** [review]

**Comment by Kharkov72** [comment]
a single second between attacks will not come up 95% of the time on NPC roombas, I'm sure

**Comment by PaulRitter** [comment]
well npc roombas weren't the problem iirc

**Comment by Blithering** [comment]
The damage should really be 1-5, it was meant to be a joke addition not a viable murder method.

**Comment by RealestEstate** [comment]
doomba2

**Comment by Kharkov72** [comment]
oh jesus oh fuck

**Pull request merged**